framework/generated/generated_vulkan_struct_encoders.cpp

```diff
@@ -5948,6 +5948,18 @@ void EncodeStruct(ParameterEncoder* encoder, const VkPipelineExecutableInternalR
 
 void EncodeStruct(ParameterEncoder* encoder, const VkPipelineLibraryCreateInfoKHR& value)
 {
+    // fix gpl issue for release build.
+#if defined(VK_USE_PLATFORM_ANDROID_KHR)
+    if (value.pNext != nullptr)
+    {
+        auto base = reinterpret_cast<const VkBaseOutStructure*>(value.pNext);
+        if (base->sType == 1)
```

Comment:
Not sure I understand what's the intention here.

This `EncodeStruct` function will be called when encoding the pNext chain of the `VkGraphicsPipelineCreateInfo`. This has already being tampered in this PR by the changes in the `CreateGraphicsPipelines()` function in `custom_vulkan_api_call_encoders.cpp`. I don't understand the purpose of the changes here.

A `VkStructureType` with the value 1 is `VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO = 1`. When encoding the pNext chain of `VkPipelineLibraryCreateInfoKHR` this would be unexpected and invalid to appear.